package.json

```diff
@@ -25,8 +30,13 @@
   "homepage": "https://github.com/voorhoede/demo-linker#readme",
   "devDependencies": {
     "commitizen": "2.8.6",
+    "csso": "2.2.1",
```

Comment:
The demo-linker is mainly for development, not so much for production. Do you feel we actually need to minify CSS & JS. If so, shall we copy the original to `dist/demo-linker.(css|js)` and add minified versions `dist/demo-linker.min.(css|js)`? I would then even like it if they had sourcemaps. But maybe we should move this to a separate PR, and focus on getting this deployed to GH Pages first.